### PR TITLE
Kylin-4037 Fix can't Cleanup Data in Hbase's HDFS Storage When Deploy Apache Kylin with Standalone HBase Cluster

### PR DESCRIPTION
**Pull request by wangxiaojing123**

### CODE REVIEW

**Comment by asfgit** [comment]
Can one of the admins verify this patch?

**Comment by asfgit** [comment]
Can one of the admins verify this patch?

**Comment by asfgit** [comment]
Can one of the admins verify this patch?

**Comment by asfgit** [comment]
Can one of the admins verify this patch?

File: server-base/src/main/java/org/apache/kylin/rest/job/StorageCleanupJob.java [hunk 1]
```diff
@@ -217,25 +217,24 @@ private void cleanUnusedHdfsFiles() throws IOException {
     
     protected void collectUnusedHdfsFiles(UnusedHdfsFileCollector collector) throws IOException {
         if (StringUtils.isNotEmpty(config.getHBaseClusterFs())) {
-            cleanUnusedHdfsFiles(hbaseFs, collector);
+            cleanUnusedHdfsFiles(hbaseFs, collector, true);
         }
-        cleanUnusedHdfsFiles(defaultFs, collector);
+        cleanUnusedHdfsFiles(defaultFs, collector, false);
```
Comment (nichunen) [inline server-base/src/main/java/org/apache/kylin/rest/job/StorageCleanupJob.java]: Why should run cleanUnusedHdfsFiles twice if config.getHBaseClusterFs() is not empty?

**Pull request merged**